firecracker.go

```diff
@@ -26,7 +26,12 @@ import (
 	ops "github.com/firecracker-microvm/firecracker-go-sdk/client/operations"
 )
 
-const firecrackerRequestTimeout = 500 * time.Millisecond
+const (
+	// env name to make firecracker request timeout configurable
+	firecrackerRequestTimeoutEnv = "FIRECRACKER_REQUEST_TIMEOUT_MILLISECONDS"
```

Comment:
For environment variables that are part of the SDK and not an application built on top of the SDK, I think it would be better to be explicit about that in the naming of the variable.  Something like "FIRECRACKER_GO_SDK_REQUEST_TIMEOUT_MILLISECONDS".